cheatsheets/gleam-for-python-users.md

```diff
@@ -614,19 +616,19 @@ Python uses the notion of exceptions to interrupt the current code flow and pop
 An exception is raised using the keyword `raise`.
 
 ```py
-def an_fn_that_fails():
-    raise Exception('an error')
+def a_fn_that_fails():
```

Comment:
"a ef en" sounds very wrong, but "an function" does as well. just depends on how you read it.

I think we should go with `a_function_that_fails` so that we don't have to argue about it. 😅

Reply:
I completely agree and updated the function name.